NotificationSystem/NotificationSystem/Template/EmailTemplate.cs

```diff
@@ -19,6 +19,27 @@ public static string GetSubscriberEmailBody(List<JObject> messages, OrcasiteHelp
             return $"<html><head><style>{GetCSS()}</style></head><body>{GetSubscriberEmailHtml(messages, orcasiteHelper)}</body></html>";
         }
 
+        public static string GetSubscriberEmailSubject(List<JObject> messages)
+        {
+            if (messages == null || messages.Count == 0)
+            {
+                return "Notification: Orca detected!";
+            }
+
+            // Extract location from first message
+            string location = null;
+            try
+            {
+                location = messages[0]["location"]?["name"]?.ToString();
+            }
+            catch
+            {
+                // If we can't get location, use null which will default to "Unknown"
+            }
```

Comment:
`GetLocation` should be a helper function as it is separate from the email subject.

Reply:
Done